presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -231,6 +232,8 @@
     public static final String PREFER_MERGE_JOIN = "prefer_merge_join";
     public static final String SEGMENTED_AGGREGATION_ENABLED = "segmented_aggregation_enabled";
     public static final String USE_EXTERNAL_PLAN_STATISTICS = "use_external_plan_statistics";
+    public static final String NODE_NUM_IN_LOGICAL_PLAN_EXCEED_THD_EXCEPTION_ENABLED = "node_num_in_logical_plan_exceed_thd_exception_enabled";
```

Comment:
We don't need this session property, if we just set max_node_num_in_logical_plan to 1e9, for example